code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -141,6 +147,21 @@
 		skipcatch = TRUE
 		blocked = TRUE
 	else if(I)
+		if (mind)
+			if (mind.martial_art && mind.martial_art.dodge_chance == 100) //autocatch for rising bass

```

Comment:
You are hindering the code ""quality"" with bad copypasta.
Can't you move this to `mob/living/carbon/can_catch_item()` and `mob/living/carbon/hitby()`, perhaps only leaving a couple lines here to stop the embedding if you wish so?

Reply:
I don't see why this would make a difference, since it would have the same effect with probably more lines of code.

Reply:
I'm not talking about modifying the game behaviour of this auto-catch-items feature, but to move it where more suitable and maybe soothe tomorrow's code ordeals just a little.